### PR TITLE
touchfix + new maxMobileScreenWidth

### PR DESCRIPTION
**Pull request by alltime84**
- Increased maxMobileScreenWidth to use mobile.css even if smartphone is turned to landcape
- added threshold for touchmove to not cancel clicks on minor moves (touch click did not react on every click)

### CODE REVIEW

**Comment by ChristianMayer** [comment]
Can you please replace the Tab characters by Space (cf. https://github.com/CometVisu/CometVisu/wiki/Coding-style)

**Comment by alltime84** [comment]
ahhhh.... OK, I hope it fits to the coding style now.

File: src/lib/templateengine.js [hunk 1]
```diff
@@ -687,7 +703,7 @@ function TemplateEngine( undefined ) {
       if (!ret && m['defaultValue']) {
         ret = mapValue(m['defaultValue']);
       }
-      if (ret) {
+      if( ret !== undefined ) {
```
Comment (ChristianMayer) [inline src/lib/templateengine.js]: As this is included here and not on the original version it seems that this patch is against a old version. (GitHub tells me that it is CometVisu/CometVisu develop but that's not correct when you refer to https://github.com/CometVisu/CometVisu/blame/develop/src/lib/templateengine.js )
=> Can you rebase that request on the latest CometVisu/CometVisu develop?

File: src/lib/templateengine.js [hunk 2]
```diff
@@ -498,7 +499,10 @@ function TemplateEngine( undefined ) {
         downtime:        0,
         alreadyCanceled: false
       };
-    
+
+    var touchStartX = null;
```
Comment (ChristianMayer) [inline src/lib/templateengine.js]: please don't use new `var` for each line. Just reuse the `var` above from line 488 ff.

File: src/lib/templateengine.js [hunk 3]
```diff
@@ -216,7 +216,8 @@ function TemplateEngine( undefined ) {
   this.bindClickToWidget = false;
     
   // threshold where the mobile.css is loaded
-  this.maxMobileScreenWidth = 480;
+	// this.maxMobileScreenWidth = 480; // modified for Menu Plugin!
+  this.maxMobileScreenWidth = 780;
```
Comment (ChristianMayer) [inline src/lib/templateengine.js]: Do you really need that? Woudn't a config file with the attribute `max_mobile_screen_width` set in the `<pages>` element be sufficient?

When you do need it: please don't keep the old version in the comment in the line above (219)

**Comment by ChristianMayer** [comment]
I've just tested it and the patch functionality seems fine to merge. 
Please check my line comments and either change the code or tell me why you think it's fine there.
Then I can merge the code.

**Comment by ChristianMayer** [comment]
@alltime84 as my pull request #242 was merged earlier and thus created a conflict for this patch, I suggest that I will resolve the collision once you've  answered my points. (Of course you are free to resolve the collision as well :))

File: src/lib/templateengine.js (hunk 3 above)
Comment (alltime84) [inline src/lib/templateengine.js]: Fine for me. No need to be hardcoded. I just did not know it exists.

File: src/lib/templateengine.js (hunk 2 above)
Comment (alltime84) [inline src/lib/templateengine.js]: Ok, feel free to change it.

**Comment by alltime84** [comment]
Please go ahead.

**Comment by ChristianMayer** [comment]
It's merged now

**Pull request merged**